ods_ci/tests/Tests/500__jupyterhub/custom-image.robot

```diff
@@ -17,7 +17,7 @@ Force Tags       JupyterHub
 *** Variables ***
 ${YAML} =         tests/Resources/Files/custom_image.yaml
 ${IMG_NAME} =            custom-test-image
-${IMG_URL} =             quay.io/thoth-station/s2i-lab-elyra:v0.1.1
+${IMG_URL} =             quay.io/opendatahub-contrib/workbench-images:jupyter-datascience-c9s-py311_2023c_latest
```

Comment:
It should be using the hash

Reply:
Better to use the tag in this case IMHO as it will keep testing an updated image without having to change the test every time